### ISSUE TITLE
npm run  dev error 

### ISSUE THREAD

**Issue by zanjs** [saltyshiomix/react-ssr#10]
```
Failed to compile.

./react-ssr-src/entry-index.jsx
Module not found: Error: Can't resolve
```

eact-ssr-jsx-starteriewsindex.jsx

**Comment by saltyshiomix** [comment]
Hi @zanjs , thank you for your reporting the issue!

Do you use it on Windows?

**Comment by zanjs** [comment]
yes !

**Comment by saltyshiomix** [comment]
@zanjs 

Fixed the issue as of `v0.13.0` 👍

**Comment by zanjs** [comment]
@saltyshiomix Nice! Thinks!
